README.md

```diff
@@ -139,6 +142,18 @@ You can also use Docker Compose to run all the components of EvalAI together. Th
     ```
     gulp dev:runserver
     ```
+   
+    * *Optional* If you get an error about missing the sass gem, you may need to also install it globally.
```

Comment:
@deshraj Would it be correct to add the above error here or in the FAQ section for developers (Issue #1130 )

Reply:
@gautamjajoo you're right. We are not gonna put error instructions with setup instructions. We should mention the recommended node version, npm version, bower version, ruby-sass etc In very clear way. And if this is known bug then we should move it to FAQ as you mentioned.